marquez_client/client.py

```diff
@@ -41,15 +46,24 @@ def __init__(self, host=None, port=None,
         self._namespace_name = namespace_name or os.environ.get(
             'MARQUEZ_NAMESPACE', DEFAULT_NAMESPACE_NAME
         )
-        self._api_base = f'http://{host}:{port}/{_API_PATH}'
+        self._api_base = f'{protocol}://{host}:{port}/{_API_PATH}'
+
+        if not port or port == 8080 or port == 80:
```

Comment:
Do we need this check? We set the port [here](https://github.com/MarquezProject/marquez-python/blob/master/marquez_client/client.py#L37) and default to port `8080` if not provided

Reply:
Also, if the goal it to make the url look cleaner when using the default port, it looks like this is not correct if port is 8080. We can only omit the port if it's 80

Reply:
removed the check for 8080.